pkg/plugins/manager/store/store.go

```diff
@@ -36,6 +36,12 @@ func New(pluginRegistry registry.Service) *Service {
 func (s *Service) Plugin(ctx context.Context, pluginID string) (plugins.PluginDTO, bool) {
 	p, exists := s.plugin(ctx, pluginID)
 	if !exists {
+		// fallback to an alias lookup
+		for _, v := range s.Plugins(ctx) {
+			if v.Alias == pluginID {
+				return v, true
+			}
+		}
```

Comment:
^^^ this is the key backend piece

Reply:
Rather than having this logic in both the store and client, it feels like this belongs in the central [registry](https://github.com/grafana/grafana/blob/main/pkg/plugins/manager/registry/in_memory.go).